Source/FieldSolver/FiniteDifferenceSolver/FiniteDifferenceAlgorithms/CartesianCKCAlgorithm.H

```diff
@@ -176,7 +177,22 @@ struct CartesianCKCAlgorithm {
                       +  F(i+1,j+1,k-1,ncomp) - F(i+1,j  ,k-1,ncomp)
                        +  F(i-1,j+1,k-1,ncomp) - F(i-1,j  ,k-1,ncomp));
 #elif (defined WARPX_DIM_XZ)
-            return 0._rt; // 2D Cartesian: derivative along y is 0
+        amrex::ignore_unused(F);
+        amrex::ignore_unused(coefs_y);
+        amrex::ignore_unused(n_coefs_y);
+        amrex::ignore_unused(i);
+        amrex::ignore_unused(j);
+        amrex::ignore_unused(k);
+        amrex::ignore_unused(ncomp);
```

Comment:
Just FYI, you can also write:
```suggestion
        amrex::ignore_unused(F, coefs_y, n_coefs_y, i, j, k, ncomp);
```

Reply:
Ah! I didn't know about that! I had never looked at how exactly `ignore_unused` is implemented. Now I've seen that it can accept an arbitrary number of arguments. I will use this feature.

Reply:
I've just scanned all the code for multiple occurrences of `amrex::ignore_unused` and I've merged them as you have suggested.